c2corg_api/tests/views/test_user.py

```diff
@@ -195,10 +196,51 @@ def test_register_forum_username_unique(self, _send_email):
         self.assertEqual(json['errors'][0]['description'],
                          'already used forum_username')
 
+    @patch('c2corg_api.emails.email_service.EmailService._send_email')
+    def test_register_username_not_email(self, _send_email):
```

Comment:
To be consistent with the following test, it should rather be `test_register_username_email_not_equals_email`